pkg/usagemetrics/googleanalytics.go

```diff
@@ -0,0 +1,79 @@
+package usagemetrics
+
+import (
+	"github.com/golang/glog"
+	analytics "github.com/jpillora/go-ogle-analytics"
+	k8sapi "github.com/openebs/maya/pkg/client/k8s/v1alpha1"
+	openebsversion "github.com/openebs/maya/pkg/version"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sversion "k8s.io/client-go/pkg/version"
+)
+
+const (
+	// GAclientID is the unique code of the project in Google Analytics
+	GAclientID = "UA-127388617-1"
+)
+
+// Event is a custom metric collected by OpenEBS developers
+type Event struct {
+	category string
+	action   string
+	label    string
+	// LabelSet bool
+	value int64
+	// valueSet bool
+}
+
+// NewMetric returns an Event struct with eventCategory, eventAction,
+// eventLabel, eventValue fields
+func NewMetric(c, a, l string, v int64) *Event {
+	return &Event{
+		category: c,
+		action:   a,
+		label:    l,
+		value:    v,
+	}
+}
+
+// PushEvent sends a single event to Google Analytics
+func (e *Event) PushEvent() error {
+	uuid, err := getUUIDbyNS("default")
+	if err != nil {
+		return err
+	}
+	gaClient, err := analytics.NewClient(GAclientID)
+	if err != nil {
+		return err
+	}
+	gaClient.ClientID(uuid)
+	// Application name and version
+	gaClient.ApplicationID("OpenEBS")
+	gaClient.ApplicationName("openebs") // Required to make an event-hit
+	gaClient.ApplicationVersion(openebsversion.GetVersion())
+	// K8s version
+	gaClient.DataSource(k8sversion.Get().GitCommit)
+	gaClient.ApplicationInstallerID(k8sversion.Get().GitCommit)
+	event := analytics.NewEvent(e.category, e.action)
+	event.Label(e.label)
+	event.Value(e.value)
+	if sendSuccessErr := gaClient.Send(event); sendSuccessErr != nil {
+		glog.Infof(string(sendSuccessErr.Error()))
+		return sendSuccessErr
+	}
+	glog.Infof("Event fired")
```

Comment:
What event gets fired? Either we remove it or put proper info.
This should be a debug IMO.